Client/trunk/ParaEngineClient/BlockEngine/BlockTessellators.cpp

```diff
@@ -644,7 +687,7 @@ void ParaEngine::BlockGeneralTessellator::TessellateStdCube(BlockRenderMethod dw
 				}
 
 				Block* pCurBlock1 = neighborBlocks[BlockCommon::NeighborLightOrder[baseIdx]];
-				if (pCurBlock1 && pCurBlock1->GetTemplate()->IsMatchAttribute(BlockTemplate::batt_solid))
+				if (pCurBlock1 && pCurBlock1->GetTemplate()->IsMatchAttribute(BlockTemplate::batt_solid) && !pCurBlock1->GetTemplate()->IsMatchAttribute(BlockTemplate::batt_invisible))
```

Comment:
can you design an API to merge the two IsMatchAttribute call into one call with bitwise operation at compile time
```
such as IsMatchAttribute(batt_solid & (^batt_invisible))
```